spec/order_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'order'
+
+describe Order do
+    it "raises error when item is not displayed in menu" do
+        expect {subject.add_item('Bread', 2).to raise_error('This item is not on the menu')}
+    end
+
+    it "can add add a item on the menu to the order array" do
+        subject.add_item('Chips', 2.00)
+        expect {subject.order.to include(["Chips", 2.00])}
```

Comment:
Again, the `expect { ... }` curly braces are in the wrong place here.

Reply:
Hi this has been corrected but for some reason when I use curly braces for this expect statement it throws an error but with () it works fine 😕